src/python_testing/TC_COMMTR_3_1.py

```diff
@@ -69,40 +66,50 @@ def desc_TC_COMMTR_3_1(self) -> str:
-            TestStep("3", "TH sends TestEventTrigger command Test Event Clear", "Status code must be SUCCESS."),
-            TestStep("4", "TH establishes a subscription to MeteredQuantity attribute.",
+            TestStep("1", "Commissioning, already done", "DUT is commissioned", is_commissioning=True),
+            TestStep("2", """Set up a subscription to the Commodity Metering cluster attributes:
+                     - MinIntervalFloor: 0
+                     - MaxIntervalCeiling: 10""",
                      "Subscription is established successfully."),
-            TestStep("5", "TH reads MeteredQuantity attribute and saves value as metered_quantity.", """
-                     - DUT replies a list of MeteredQuantityStruct entries;
-                     - Value is saved as metered_quantity."""),
-            TestStep("6", "TH sends TestEventTrigger command Fake Value Update Test Event", "Status code must be SUCCESS."),
-            TestStep("7", "TH awaits a ReportDataMessage containing MeteredQuantity attribute.", """
-                     - Verify that the report on change is received;
-                     - Value does not match the metered_quantity value."""),
-            TestStep("8", "TH reads MeteredQuantity attribute and saves the initial value as saved.", """
-                     - DUT replies a list of MeteredQuantityStruct entries;
-                     - Value is saved as metered_quantity."""),
-            TestStep("9", "TH sends TestEventTrigger command Fake Value Update Test Event", "Status code must be SUCCESS."),
-            TestStep("10", "TH awaits a ReportDataMessage containing MeteredQuantity attribute.", """
-                     - Verify that the report on change is received;
-                     - Value does not match the metered_quantity value."""),
-            TestStep("11", "TH reads MeteredQuantity attribute and saves the initial value as saved.", """
-                     - DUT replies a list of MeteredQuantityStruct entries;
-                     - Value is saved as metered_quantity."""),
-            TestStep("12", "TH sends TestEventTrigger command to General Diagnostics Cluster for Update Test Event Clear",
+            TestStep("3", "TH reads MaximumMeteredQuantities attribute.", """
+                     - DUT replies a uint16 value or null;
+                     - Value is saved as MaxMeteredQuantitiesValue."""),
+            TestStep("4", "TH reads MeteredQuantity attribute.", """
+                     - DUT replies a list of MeteredQuantityStruct entries or null;
+                     - Value is saved as MeteredQuantityValue."""),
+            TestStep("5", "TH reads MeteredQuantityTimestamp attribute.", """
+                     - DUT replies a epoch-s value, or null;
+                     - Value is saved as MeteredQuantityTimestampValue."""),
+            TestStep("6", "TH reads TariffUnit attribute.", """
+                     - DUT replies a TariffUnitEnum value or null;
```

Comment:
Nit: text here not exactly per latest test plan: `DUT replies a TariffUnitEnum value in range 0-1, or null;`
```suggestion
                     - DUT replies a TariffUnitEnum value in range 0-1, or null;
```